extension/manifest.json

```diff
@@ -205,10 +204,9 @@
 				"scripts/features/total-item-cost/ttTotalItemCost.js",
 				"scripts/features/bazaar-worth/ttBazaarWorth.js",
 				"scripts/features/item-redirect/ttBazaarRedirect.js",
-				"scripts/features/bazaar-worth/ttBazaarWorth.js",
 				"scripts/features/bazaar-fill-max/ttBazaarFillMax.js"
 			],
-			"run_at": "document_end"
+			"run_at": "document_start"

```

Comment:
This is something you absolutely shouldn't do. Most features rely on them being ran at `document_end`. If you need one to run at `document_start`: 1. rename it to `xxx.entry.js`, to indicate the run time 2. put it in it's own section with `document_start`.